validateOffload.py

```diff
@@ -102,7 +104,8 @@ def stat(self, duration: int) -> Result:
             r1 = self.run_ethtool_cmd(self.ethtool_cmd)
             if r1.returncode != 0:
                 return r1
-            time.sleep(duration)
```

Comment:
Good catch during the presentation. I really like your fix here :+1:

Reply:
Thank you :wink: